package.json

```diff
@@ -5,52 +5,33 @@
   "author": "Antony Jones",
   "license": "MIT",
   "scripts": {
-    "dev": "vite dev",
-    "build": "vite build",
+    "dev": "vite dev --host",
+    "build": "concurrently pnpm:build:*",
+    "build:app": "vite build",
+    "build:sdk": "vite build --config vite.components.config.js",
     "preview": "vite preview",
-    "package": "svelte-kit sync && svelte-package && publint",
-    "prepublishOnly": "npm run package",
     "check": "svelte-kit sync && svelte-check --tsconfig ./jsconfig.json",
     "check:watch": "svelte-kit sync && svelte-check --tsconfig ./jsconfig.json --watch",
     "lint": "eslint .",
     "test": "mocha \"src/**/*.spec.js\""
   },
-  "exports": {
-    ".": {
-      "types": "./dist/index.d.ts",
-      "svelte": "./dist/index.js"
-    },
-    "./Banner.svelte": "./dist/Banner.svelte",
-    "./style.css": "./dist/style.css"
-  },
   "files": [
-    "dist",
-    "!dist/**/*.spec.*"
+    "src/lib/sdk",
+    "static/v1/sdk"
```

Comment:
This would probably still be dist, since we are just building a regular package.